apps/condo/domains/marketplace/components/MarketplaceOnboarding/OfferSetup.tsx

```diff
@@ -65,48 +69,52 @@ export const OfferSetup: React.FC<{ launchContext: LaunchContextType }> = ({ lau
     const {
         obj: organization,
         loading: organizationIsLoading,
-    } = Organization.useObject(
-        { where: { id: organizationId } }
-    )
+    } = Organization.useObject({ where: { id: organizationId } })
+
+    const {
+        obj: acquiringOffer,
+        loading: isAcquiringOfferLoading,
+    } = Accept.useObject({ where: { scope: SCOPE_TYPES.acquiring, organizationId }, sortBy: [SortAcceptsBy.CreatedAtDesc], first: 1 })
```

Comment:
IMO useObject already returns single value, so you don't need to filter with first:1 at the options prop

Reply:
`useObject` will throw an error in the case of 2+records. I just need to load the last created object.